docs/high-availability/etcd.manifest

```diff
@@ -3,63 +3,56 @@
 "kind": "Pod",
 "metadata": {"name":"etcd-server"},
 "spec":{
-"hostNetwork": true,
-"containers":[
+    "hostNetwork": true,
+    "containers":[
     {
     "name": "etcd-container",
-    "image": "gcr.io/google_containers/etcd:2.0.9",
+    "image": "quay.io/coreos/etcd",
     "command": [
-              "/usr/local/bin/etcd",
-              "--name", "${NODE_NAME}",
-              "--initial-advertise-peer-urls",
-              "http://${NODE_IP}:2380",
-              "--listen-peer-urls",
-              "http://${NODE_IP}:2380",
-	      "--advertise-client-urls",
-              "http://${NODE_IP}:4001",
-              "--listen-client-urls",
-	      "http://127.0.0.1:4001",
-              "--data-dir",
-	      "/var/etcd/data",
-              "--discovery",
-              "${DISCOVERY_TOKEN}"
-            ],
+          "/etcd","--name","NODE_NAME",
+	      "--initial-advertise-peer-urls",  "http://NODE_IP:2380",
+	      "--listen-peer-urls",     	"http://NODE_IP:2380",
+	      "--advertise-client-urls",    	"http://NODE_IP:2379",
+	      "-initial-cluster", "kube0.ha=http://192.168.4.100:2380",
+	      "--listen-client-urls",   	"http://127.0.0.1:2379,http://NODE_IP:2379",
+          "--data-dir","/var/etcd/data"
+    ],
     "ports":[
-      { "name": "serverport",
-        "containerPort": 2380,
-        "hostPort": 2380},{
-       "name": "clientport",
-        "containerPort": 4001,
-        "hostPort": 4001}
-        ],
+        {
+            "name": "serverport", "containerPort": 2380, "hostPort": 2380
+        },
+        {
+            "name": "clientport","containerPort": 4001, "hostPort": 4001
+        }
+    ],
     "volumeMounts": [
         { "name": "varetcd",
         "mountPath": "/var/etcd",
         "readOnly": false},
         { "name": "etcssl",
         "mountPath": "/etc/ssl",
-        "readOnly": true},
+        "readOnly": false},
         { "name": "usrsharessl",
         "mountPath": "/usr/share/ssl",
-        "readOnly": true},
+        "readOnly": false},
         { "name": "varssl",
         "mountPath": "/var/ssl",
-        "readOnly": true},
+        "readOnly": false},
         { "name": "usrssl",
         "mountPath": "/usr/ssl",
-        "readOnly": true},
+        "readOnly": false},
         { "name": "usrlibssl",
         "mountPath": "/usr/lib/ssl",
-        "readOnly": true},
+        "readOnly": false},
         { "name": "usrlocalopenssl",
         "mountPath": "/usr/local/openssl",
-        "readOnly": true},
+        "readOnly": false},
         { "name": "etcopenssl",
         "mountPath": "/etc/openssl",
-        "readOnly": true},
+        "readOnly": false},
         { "name": "etcpkitls",
         "mountPath": "/etc/pki/tls",
-        "readOnly": true}
+        "readOnly": false}
```

Comment:
i modified this because i was trying to get certs right and it made that process a little easier.  iirc its not required to be false for any reason,  i can revert it to true but i figured most folks will just customize this whole block either way.